guides/common/modules/proc_running-installer.adoc

```diff
@@ -3,6 +3,12 @@
 [id="running-the-{project-context}-installer_{context}"]
 = Running the {Project} installer
 
+The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup.
+It uses native operating system packaging (`.rpm` or `.deb` packages) and adds necessary configuration for the complete installation.
+
+Components include the Foreman web UI, Smart Proxy, a Puppet server, TFTP, DNS and DHCP servers.
+The installer is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
+
 The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `{foreman-installer} --help`, or by running `{foreman-installer} -i` for interactive mode.
 More examples are described in the `Installation Options` section.
 The `-v` option disables the progress bar and displays all changes.
```

Comment:
If the goal is to keep the Quickstart guide as short as possible, I think we could consider at least some of this to be implementation details and drop some lines. I welcome suggestions on what to drop.